composer.json

```diff
@@ -13,7 +13,7 @@
   "license": "Apache-2.0",
   "authors": [],
   "require": {
-    "psr/log": "~1.0",
+    "psr/log": "^1.0.0 || ^3.0.0",
```

Comment:
This sdk should be installable as dependency for applications on php8 with `psr/log 3`